ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,39 +32,205 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
-    }
+  function  assertAddTermCommon(assert, rootNode) {
+    //Common tests for Add terms, use in term and taxonomy components
```

Comment:
You should use [JSDoc](http://usejsdoc.org/about-getting-started.html) for these kinds of comments